.github/workflows/test.yaml

```diff
@@ -0,0 +1,46 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [windows-2019, windows-latest]
+    
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    # Environment requirements:
+    # https://learn.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment
+    - name: Test docker
+      run: |
+        docker version
+        docker pull hello-world
+        docker run hello-world
```

Comment:
don't think we need to run this step every time we run the workflow, since we know Docker will be part of the GH runner (based on Harrison's comment) we know this step should work.